docs/scripts.rst

```diff
@@ -130,6 +130,10 @@ Explanation for the keys:
 - ``urls``: dictionary of table names and the respective urls
 - ``version``: "1.0.0"
 
+``fields``: Although retriever automatically detects the schema of the table,
```

Comment:
You are correct. Pk-auto would change the value of tree_id from char to int. I think this is important to mention, so putting it in the doc would be helpful. Also, I'm not 100% sure that pk-auto is working correctly. I changed one of the fields from a script to pk-auto, and it overwrote the column but also changed some other columns. It might be worth looking into the code for pk-auto if you can.